java/client/src/org/openqa/selenium/remote/mobile/RemoteNetworkConnection.java

```diff
@@ -42,7 +42,7 @@ public ConnectionType getNetworkConnection() {
   @Override
   public ConnectionType setNetworkConnection(
       ConnectionType type) {
-    Map<String, ConnectionType> mode = ImmutableMap.of("type", type);
+    Map<String, Integer> mode = ImmutableMap.of("type", type.getBitMask());
```

Comment:
can you change this instead to just `type.toString()` and then you wouldn't have to expose the getBitMask in the enum. (Alternatively you could have used `type.hashCode()` but that doesn't feel as nice)